tests/base.py

```diff
@@ -20,6 +20,7 @@ class TwitterAds(unittest.TestCase):
     INCREMENTAL = "INCREMENTAL"
     OBEYS_START_DATE = "obey-start-date"
     PAGE_SIZE = 1000
+    account_id = ""
```

Comment:
Account_Id will be string or int?

Reply:
`account_id` will be a string. Actually, we expect the comma-separated list of account ids in the config.